fedimint-api/src/config.rs

```diff
@@ -71,8 +72,10 @@ impl FederationId {
         Self(threshold_crypto::PublicKey::from(G1Projective::identity()))
     }
 
-    fn from_bytes(bytes: [u8; 48]) -> Self {
-        Self(threshold_crypto::PublicKey::from_bytes(bytes).expect("Invalid pub-key"))
+    fn from_bytes(bytes: [u8; 48]) -> Result<Self> {
```

Comment:
I was wondering if I should pull in `anyhow::Result` or just use `Option` since there's really on case.